impersonation/README.md

```
@@ -0,0 +1,92 @@
+    - **Content:** `{ "statusCode": 403, "error": "Forbidden", "message": "Request already exists please wait for approval or rejection"`
+  - **Code:** 403
+    - **Content:** `{ "statusCode": 403, "error": "Forbidden", "message": "Please complete impersonation before creating a new request"`
+  - **Code:** 500
+    - **Content:** `{ "statusCode": 500, "error": "Internal Server Error", "message": "An internal server error occurred" }`
+  - **Code:** 404
+    - **Content:** `{ "statusCode": 404, "error": "Not Found", "message": "User not found" }`
+  - **Code:** 401
+    - **Content:** `{ "statusCode": 401, "error": "Unauthorized", "message": "You are not authorized for this action." }`
+
+
+#### Authentication and Authorization:
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)